todo_app/app.py

```diff
@@ -1,11 +1,18 @@
-from flask import Flask
+from flask import Flask, render_template, redirect, request
 
 from todo_app.flask_config import Config
-
+from todo_app.data.session_items import get_items,add_item
 app = Flask(__name__)
 app.config.from_object(Config())
 
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    items = get_items()
+    return render_template('index.html', items=items)
+
+@app.route('/add', methods=["POST"])
+def post_item():
+    title = request.form["title"]
+    add_item(title)
+    return redirect("http://127.0.0.1:5000",302)
```

Comment:
This works, but specifying that we're redirecting to localhost/127.0.0.1 is going to bite us if/when we try to deploy this - Flask will actually default the redirects to the same host url the request came from to help with this, so we can shift this for:
```suggestion
    return redirect("/", 302)
```